frontend/src/index.css

```diff
@@ -11,3 +11,170 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.headerArea {
+  /* margin-top: 1em; */
+  margin-bottom: 1em;
+  border-bottom: 1px solid #d3d3d3;
+}
+
+input {
+  padding: 0.5em;
+}
+select {
+  padding: 0.5em;
+}
+
+.tableCell {
+  padding: 1px 2em;
+  font-size: 0.875rem;
+  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
+  font-weight: 400;
+  line-height: 1.43;
+  vertical-align: inherit;
+}
+
+.numeric {
+  text-align: right;
+}
+.tableHeadCell {
+  padding: 1em;
+  text-align: left;
+  border-bottom: 1px solid rgba(224, 224, 224, 1);
+  border-top: 1px solid rgba(224, 224, 224, 1);
+}
+
+.tableHeadCell input{
+  padding: .5em;
+  text-align: left;
+  background-color: white;
+}
+
+#statusSearch{
+  padding: .5em;
+}
+#sortBySelect{
+  padding: 0.5em 2em .5em .5em;
+  width: max-content;
+}
+
+.SearchField{
+  margin: 0em 2em 0em 0em;
+}
+.SearchField input{
+  width: 55em;
```

Comment:
Maybe make the width larger when taking into account the suggestion to align the sort by to the right.

Reply:
honestly i'm not a fan of having a fixed width for much..... if the screens aren't wide enough it ends up looking like this

<img src="https://user-images.githubusercontent.com/22307286/97240601-126c0f80-17c5-11eb-99e2-f689187e6e91.png" width="50%" />

although now that i check this more closely... it's a problem that applies to other elements just as easily.

`63em` makes it fit quite nicely on my screen, can you confirm this is a good size for you too before i commit the change @sharon-ho ? :joy:

Reply:
It's a bit too big when selecting "Last Access", I probably recommend 60em. Though that's also subject to change depending on what the max length of one of the dropdown selections for "Sort by".

Reply:
i think this is another thing that should just be moved to bug report and left alone for the moment. it doesn't really impact functionality that much so it can be polished later since it isn't a quick fix.

Reply:
Bug report for the Sort by: #107
Bug report for the Search bar: #108